.travis.yml

```diff
@@ -81,6 +81,7 @@ before_cache:
   - find $HOME/.sbt -name "*.lock" -delete
 
 notifications:
+  email: play-team@lightbend.com
```

Comment:
Didn't gmail block incoming e-mails because posting on the group is restricted to participants in the group?

Reply:
I see we use the group in https://github.com/playframework/playframework/blob/master/.travis.yml#L89 too but I no longer remember if that works either.

Reply:
I think that problem might be specific to our benchmarking server sending emails, not for Travis CI sending emails.